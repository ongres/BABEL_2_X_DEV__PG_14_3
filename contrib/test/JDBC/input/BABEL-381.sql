-- BABEL-381 Test numeric constant can be correctly processed through
-- TDS protocol.
select 2.0;
go

select 2.0/1.5;
go

select 2.0, 2.0/1.5, 1.0/1.5;
go
