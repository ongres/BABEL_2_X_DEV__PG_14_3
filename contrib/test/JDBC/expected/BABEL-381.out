-- BABEL-381 Test numeric constant can be correctly processed through
-- TDS protocol.
select 2.0;
go
~~START~~
numeric
2.0
~~END~~


select 2.0/1.5;
go
~~START~~
numeric
1.3333333333333333
~~END~~


select 2.0, 2.0/1.5, 1.0/1.5;
go
~~START~~
numeric#!#numeric#!#numeric
2.0#!#1.3333333333333333#!#0.66666666666666666667
~~END~~

